internal/hcsoci/hcsdoc_wcow.go

```diff
@@ -479,6 +501,23 @@ func parseAssignedDevices(ctx context.Context, coi *createOptionsInternal, v2 *h
 	return nil
 }
 
+func parseDumpCount(annots map[string]string) (int32, error) {
```

Comment:
you could probably also get away with exporting `"internal\oci\annotations.go".parseAnnotationsUint64` and using that here

Reply:
The logAnnotationParseError() function that parseAnnotationsUint64() uses does not seem to be accurate for this case as this isn't an OCI annotation.

Reply:
lol I was distracted by a meeting and totally misworded what I wanted to say - 
The logAnnotationParseError() function that parseAnnotationsUint64() uses does not seem to be accurate for this case. I think a more desirable behavior would be to return error if value passed in is invalid and then fail container creation instead of defaulting dumpCount regkey to 'def' in parseAnnotationsUint64(). When we silently default to some value, user may report bugs saying they set a value which we aren't honoring.

Reply:
ahh, gotcha
that makes sense